crates/cast/src/main.rs

```diff
@@ -65,9 +65,17 @@ struct Cli {
     json: bool,
 
     /// If passed, command will wait until transaction is accepted or rejected
-    #[clap(short, long)]
+    #[clap(short = 'w', long)]
     wait: bool,
 
+    /// Adjusts the time after which --wait assumes transaction was not received or rejected
+    #[clap(short = 't', long)]
+    wait_timeout: Option<u16>,
+
+    /// Adjusts the time between consecutive attempts to fetch transaction by --wait flag
+    #[clap(long)]
+    wait_retry_interval: Option<u8>,
```

Comment:
not really, I just figured this is probably not something users will tinker with either way (at least not using cli, maybe rather cast config).

Reply:
I thought it might be reasonable to have it just to be consistent with `wait_timeout`, but on the other thought, it's probably gonna be used significantly less frequently

Reply:
I think we should have either short forms for both flags or for neither of them

Reply:
k, changed